content/en/user-guide/lambda-tools/hot-reloading/index.md

```diff
@@ -385,9 +378,10 @@ Next, you can build the Lambda function:
 $ yarn run build
 {{< / command >}}
 
-The `build` script in the `package.json` file uses Nodemon to watch for changes in the `src` directory and rebuild the Lambda. This is enabled using the [`nodemon-webpack-plugin`](https://www.npmjs.com/package/nodemon-webpack-plugin)  plugin, which has been pre-configured in the `webpack.config.js`  file.
+The `build` script in the `package.json` file uses Nodemon to watch for changes in the `src` directory and rebuild the Lambda.
+This is enabled using the [`nodemon-webpack-plugin`](https://www.npmjs.com/package/nodemon-webpack-plugin)  plugin, which has been pre-configured in the `webpack.config.js`  file.
 
-##### Creating the Lambda Function
+##### Creating the Lambda Function with Webpack
```

Comment:
Any particular reason for adding Webpack here? I think the process is same, irrespective of which library we are using 🤔

Reply:
This makes it easier to see where you are in the documentation.